spec.html

```diff
@@ -41915,40 +41915,40 @@ <h2>Syntax</h2>
     <emu-annex id="sec-regular-expressions-patterns">
       <h1>Regular Expressions Patterns</h1>
       <p>The syntax of <emu-xref href="#sec-patterns"></emu-xref> is modified and extended as follows. These changes introduce ambiguities that are broken by the ordering of grammar productions and by contextual information. When parsing using the following grammar, each alternative is considered only if previous production alternatives do not match.</p>
-      <p>This alternative pattern grammar and semantics only changes the syntax and semantics of BMP patterns. The following grammar extensions include productions parameterized with the [U] parameter. However, none of these extensions change the syntax of Unicode patterns recognized when parsing with the [U] parameter present on the goal symbol.</p>
+      <p>This alternative pattern grammar and semantics only changes the syntax and semantics of BMP patterns. The following grammar extensions include productions parameterized with the [UnicodeFlag] parameter. However, none of these extensions change the syntax of Unicode patterns recognized when parsing with the [UnicodeFlag] parameter present on the goal symbol.</p>
       <h2>Syntax</h2>
       <emu-grammar type="definition">
         Term[U, N] ::
```

Comment:
```suggestion
        Term[UnicodeFlag, N] ::
```
There are 12 other occurrences of "U" in this `<emu-grammar>` that need to be converted.

Reply:
Good catch; done!